src/app/query/bitcoin/runes/runes.hooks.ts

```diff
@@ -0,0 +1,21 @@
+import { createMoney } from '@shared/models/money.model';
+
+import type { RuneToken } from '../bitcoin-client';
+import { useGetRunesOutputsByAddressQuery } from './runes-outputs-by-address.query';
+import { useGetRunesWalletBalancesQuery } from './runes-wallet-balances.query';
+
+export function useRuneTokens(address: string) {
+  return useGetRunesWalletBalancesQuery(address, {
+    select: resp =>
+      resp.map(rune => {
+        return {
+          ...rune,
+          balance: createMoney(Number(rune.total_balance), rune.rune_name, 0),
+        } as RuneToken;
+      }),
```

Comment:
It'd be a more type safe approach to avoid the `as` and use a `makeRuneToken` function that returns `RuneToken`.

Internally, this fn can check the values are correct at runtime

Reply:
Thx for addressing @alter-eggo , will resolve this. 👍